### ISSUE TITLE
mp3 rips are monaural (when built from master)

### ISSUE THREAD

**Issue by Skyr** [cyanreg/cyanrip#49]
I just compiled cyanrip from master's HEAD (daf8161707a9f7b4c73b62a3e2865b46c3c65a0e)
Ripping a cd using e.g. `cyanrip -o mp3 -b 192 -s 667` results in mono mp3 files:
```
$ file 01.mp3
01.mp3: Audio file with ID3 version 2.4.0, contains: MPEG ADTS, layer III, v1, 192 kbps, 44.1 kHz, Monaural
```
I tried with a different format (FLAC), which gave stereo files as expected.

I retried compiling cyanrip from tag v0.9.0 which works fine.

Tested on Fedora 38.

**Comment by cyanreg** [comment]
Thanks, fixed by 0383d4bb319740ae74739bb3de69a65a045c66b8

**Comment by Skyr** [comment]
Wow, _that's_ a quick response - thanks for fixing!
And, btw, thanks for the cool tool, it's my absolute go-to solution for archiving my cd collection!
